### ISSUE TITLE
Ack opens quickfix also when no results found

### ISSUE THREAD

**Issue by mboughaba** [mileszs/ack.vim#225]
Hi,

Ack always opens quickfix window even if no results are matched.
I reproduced this with Ag silver searcher as Ack command.

**Comment by ches** [comment]
You're right, and it seems `:grep` and `:vimgrep` avoid this so I'd call it an ack.vim bug.

Note that there isn't a good way I could find to avoid this happening when using vim-dispatch support, but it shouldn't happen with ordinary synchronous searches.

**Comment by ches** [comment]
Related: #167.
